ec2/asg/asg.tf

```diff
@@ -44,6 +44,11 @@ variable "instance_volume_size" {
   default     = "8"
 }
 
+variable "docker_storage" {
```

Comment:
This variable name does not make sense; `ec2/asg` is a generic module and should not _always_ mount a new volume on the instance, and if it does it should not be called `docker_*` (because it's not really specific to docker). 

If we need the ability to mount new EBS drives, I suppose `ec2/asg` would have to accept a list of maps with the `ebs_block_device` configurations. Or you can go for a single map which creates a single drive if that is the only use case we have so far.

Reply:
The first idea was to use the storage only if it is needed for ECS, that's why the name and also the name of device `/dev/xvdcz`, it's hardly used for any other use-case.

I'll go with the map of settings and see. `ebs_block_device` does not support count, so I am not sure how to use it with a list.